cleanlab/internal/validation.py

```diff
@@ -18,12 +18,18 @@
 Checks to ensure valid inputs for various methods.
 """
 
+from typing import Any, List, Optional
 import warnings
 import numpy as np
 import pandas as pd
 
 
-def assert_valid_inputs(X, y, pred_probs=None, multi_label=False):
+def assert_valid_inputs(
+    X: np.ndarray,
+    y: np.ndarray,
```

Comment:
the possible valid types for `y` would include: 
list, np.ndarray, np.generic, pd.Series, pd.DataFrame

Reply:
mypy isn't happy about using `len(y: np.generic)`, as it's undefined.

Reply:
I'll start with everything except np.generic for now.

I'm trying not to change anything in the function bodies to avoid breaking any assumptions that are made in the rest of the codebase.